### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
- Add styles to the books route.
- Add styles to the categories route.

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -15,16 +17,21 @@ const Navbar = () => {
...
+                {link.text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="avatar">
+        <FaUserAlt />
+      </div>
     </nav>
```
Comment (thecodechaser) [inline src/components/Navbar.jsx]: - [ ] Your design is missing `Bookstore CMS` title from the design. Please make sure to add that and it should match with the zeplin design.

### Your design

image

### Zeplin design

image

File: src/components/Chapter.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Chapter = () => (
+  <div className="chapter-info">
+    <div>
+      <p className="title position">Current Chapter</p>
+      <p className="chapter-name">Chapter 23</p>
+    </div>
+    <button type="button">Update Progress</button>
+  </div>
```
Comment (thecodechaser) [inline src/components/Chapter.jsx:10]: - [ ] Please reduce the height of the `Update progress` button and make sure it matches with the zeplin design.

### Your design

image

### Zepline design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Fadahunsiseyi👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] Please change the title of your project in the README file and use the correct title, Right now it is `Todo Application`. I think you forgot to change it.

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @Fadahunsiseyi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**